portal/src/main/scala/cromwell/pipeline/datastorage/dao/entry/ProjectEntry.scala

```diff
@@ -12,9 +12,10 @@ trait ProjectEntry {
     def projectId = column[ProjectId]("project_id", O.PrimaryKey)
     def ownerId = column[UserId]("owner_id")
     def name = column[String]("name")
+    def description = column[String]("description")
```

Comment:
Don't forget to make a migration that will add this column to a DB table.

Reply:
@myazinn I'm keeping this in mind. Cause ProjectService test requires an additional changeset with the description column otherwise it fails